target_game.py

```diff
@@ -23,23 +26,119 @@ def move_target_at_random_pos(target):
     new_pos = (pos_x, pos_y)
     target.setPos(new_pos)
     
+
+# Now I create a function that will draw 3 targets - each for each level of the game
+# On these targets I want to show where the player clicked (scored) on each level
+def draw_overview_target(target_x, target_y, mouse_clicks):
+    target = ImageStim(win, 'target.png', size = (420, 420), pos=(target_x, target_y))
+    target.draw()
+    
+    for target_hit_pos in mouse_clicks:
+        circle = Circle(win, radius=5, fillColor='yellow', lineColor='yellow')
+        circle.setPos((target_hit_pos['mouse_x']+target_x, target_hit_pos['mouse_y']+target_y))
+        circle.draw()
+
+# Here I create a function for the intro, which allows me to explain the game to the participant
+def show_intro(win):
+    start = TextStim(
+        win,
+        text = "Hi there, player! This is you! Let's play!",
+        height = 35,
+        color = (0.2,0.2,0.8),
+        pos = (250,230),
+    )
+    
+    player = ImageStim(
+        win,
+        'creature.jpg',
+        pos = (0,-100),
+        size=(420,420),
+    )
+    
+    intro_text = (
+        'In this game you should shoot at the target with your mouse. '
+        + 'The target moves rapidly to a new position each time you shoot at it.' 
+        + 'There are 3 levels each faster than the previous one. Good luck!'
+    )
+    
+    instructions = TextStim(
+        win,
+        text = intro_text,
+        height = 35,
+        color = (1, 0.2, 0.6),
+        pos = (250, 0),
+    )
+    
+    start.draw()
+    player.draw()
+    win.flip()
+    wait(3)
+    instructions.draw()
+    win.flip()
+    wait(8)           
+    
+    
 # I call out the objects I will use and define my DISP and BGC:
-DISPSIZE = (1400,800)
+DISPSIZE = (1400, 800)
+# DISPSIZE = (GetSystemMetrics(0),GetSystemMetrics(1)) we can use this later on,
+# but for now we need to build the programme
 BGC = (-1,-1,-1)
+
+
 score = 0
+lives =  3
+level = 1
+mouse_x = 0
+mouse_y = 0
 
 win = Window(size=DISPSIZE, units='pix', fullscr=False, color=BGC)
 target = ImageStim(win, 'target.png', size = (420, 420)) # i created this target in photoshop
 mouse = Mouse(win)
 
+
+
+lives_count = TextStim(win, text = f'Lives = {lives}', height = 35, color = (1,0.2,0.6), pos = (-50,330))
+score_count = TextStim(win, text = f'Score = {score}', height = 35, color = (0.2,0.2,0.8), pos = (450,330))
+level_count = TextStim(win, text = f'Level = {level}', height = 35,color = (1, -0.5, 1), pos=(850, 330))
+
+
+
+#we need this for the overview at the end of the game
+target_hits_per_level = [
+    [],

```

Comment:
Isnt it enough to do target_hits_per_level = [] and then it will add the items of a list with a function like numpy.append or so?